brambling/views/utils.py

```diff
@@ -225,3 +228,77 @@ def get_context_data(self, **kwargs):
             'next_step': self.current_step.next_step if self.current_step else None,
         })
         return context
+
+
+class FinanceTable(object):
+
+    def __init__(self, event, transactions):
+        self.event = event
+        self.transactions = transactions
+
+    def headers(self):
+        return [
+            'Timestamp (%s)' % self.event.timezone,
+            'Created By',
+            'Method',
+            'Type',
+            'Order',
+            'Amount',
+            'Dancerfly Fee',
+            'Payment Fee',
+        ]
+
+    def header_row(self):
+        return [self.headers()]
+
+    def plaintext_rows(self):
+        yield self.headers()
+        for transaction in self.transactions:
+            yield [c.value for c in self.format_transaction(transaction)]
+
+    def cell_rows(self):
+        for transaction in self.transactions:
+            yield self.format_transaction(transaction)
+
+    def transaction_rows(self):
+        return islice(self.all_rows(), 1, None)
```

Comment:
`all_rows` doesn't actually exist, though I think I saw it in a previous iteration. Pretty sure this method is obviated.